frontend/src/sharedComponents/Metadata/ContentComps.jsx

```diff
@@ -121,8 +121,31 @@ const StarEmptyStyled = styled(StarEmpty)`
 
 const ContentCollapsed = ({ metadata, showCategory, showMunicipality }) => {
   const {
-    uuid, description, releaseState, municipalityName, metadataTypeName,
+    uuid, description, releaseState, municipalityName, metadataTypeUuid,
   } = metadata;
+  const [metadataTypeName, setMetadataTypeName] = useState('');
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const getMetadataTypeName = async () => {
+      try {
+        const res = await fetch(`/api/MetadataType/${metadataTypeUuid}`);
+        if (res.status === 200) {
+          const { name } = await res.json();
+          setMetadataTypeName(name);
+        }
+      } catch (err) {
+        const { status } = err;
+        if (status === 404) {
+          dispatch(alertActions.error('Could not fetch metadata type name'));
```

Comment:
Could not find may be more specific in this case. Could not fetch indicates something else might have happened in the chain.
```suggestion
          dispatch(alertActions.error('Could not find metadata type name'));
```